.github/actions/setup-etcd/action.yml

```diff
@@ -0,0 +1,45 @@
+name: 'Setup etcd'
+description: 'Setup tarantool environment and add it to the PATH'
+inputs:
+  etcd-version:
+    description: 'Release name from https://github.com/etcd-io/etcd/releases'
+    required: true
+    default: v3.4.13
+  install-prefix:
+    description: 'Where to extract the archive'
+    default: './etcd'
+  export-path:
+    description: 'Whether to export PATH environment variable'
+    default: false
+runs:
+  using: 'composite'
+  steps:
+    - shell: bash
+      env:
+        BASE_URL: "https://github.com/etcd-io/etcd/releases/download"
+        ETCD_VER: ${{ inputs.etcd-version }}
+        INSTALL_PREFIX: ${{ inputs.install-prefix }}
+      run: >
+        set -x;
+        mkdir -p ${INSTALL_PREFIX}
+
+        curl -L ${BASE_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz |
+        tar xz --strip-components=1 -C ${INSTALL_PREFIX}
+        etcd-${ETCD_VER}-linux-amd64/etcdctl
+        etcd-${ETCD_VER}-linux-amd64/etcd
+
+    - shell: bash
+      env:
+        INSTALL_PREFIX: ${{ inputs.install-prefix }}
+      run: |
+        set -x
+        ${INSTALL_PREFIX}/etcd --version
+        ${INSTALL_PREFIX}/etcdctl version
+
+    - shell: bash
+      run: |
+        if ( "${{ inputs.export-path }}" == "true" ); then
```

Comment:
Why do you need this "if"? Is this flow not deterministic?

Reply:
You're right, it's obsolete.